validate.rb

```diff
@@ -0,0 +1,36 @@
+require "json_schemer"
```

Comment:
Should we go ahead and add a Gemfile for this (and any future libraries)?

Reply:
good thinking, I've submitted another commit with that in